### ISSUE TITLE
Historical data

### ISSUE THREAD

**Issue by pparshakov** [valentinumbach/SoFIFA#2]
Thank you for the package!
Is it possible to get historical data on player's skills?

**Comment by valentinumbach** [comment]
No, I haven't implemented the scraping of historical data (only current scores). This should be possible, but needs more efforts.
However, as stated in https://github.com/valentinumbach/SoFIFA/issues/1, the scraper is currently broken anyway because of changes to the sofifa.com website. If you are able to contribute to a solution, pull requests are greatly appreciated :)
